content/installation/_index.md

```diff
@@ -51,12 +51,25 @@ If your company's primary domain is `example.com` you can consider a domain like
 
 1. Fill in the Domain for automated DNS configuration (see the screenshot below)
 1. Scroll down to the bottom of the page and tick the boxes acknowledging that CloudFormation will create IAM resources such as IAM Roles and IAM Policies
-1. Click the **Create Stack** button and it will start the installation!
+1. Click "Create changeset" (creating a changeset instead of "create stack" allows you to enable options that protect against errors that can ocassionally occur on stack rollbacks)
```

Comment:
Does the changeset have to be named? The screenshot shows "superwerker-example". Did you set this on purpose?

Reply:
I just set it because we use example for the domain or superwerker-xyz in other screenshots. I can revert it to the default or leave it out if that's preferable